src/JSInterop/Microsoft.JSInterop.JS/src/src/Microsoft.JSInterop.ts

```diff
@@ -244,9 +260,10 @@ export module DotNet {
      * @param methodIdentifier The identifier of the method to invoke. The method must have a [JSInvokable] attribute specifying this identifier.
      * @param dotNetObjectId If given, the call will be to an instance method on the specified DotNetObject. Pass null or undefined to call static methods.
      * @param argsJson JSON representation of arguments to pass to the method.
-     * @returns JSON representation of the result of the invocation.
+     * @param byteArrays Byte array data extracted from the arguments for direct transfer.
+     * @returns SerializedArgs containing the string JSON args along with the extracted byte arrays representation of the result of the invocation.
```

Comment:
Would it be OK to tweak the naming here? Since this is a return value, calling it "args" is quite confusing. It would be great to declare another type like `SerializedResult`, even if structurally it's the same as `SerializedArgs`.

Reply:
Same with `invokeJSFromDotNet` below.